src/sentry/models/apikey.py

```diff
@@ -44,6 +44,7 @@ class ApiKey(Model):
             ("org:read", "org:read"),
             ("org:write", "org:write"),
             ("org:admin", "org:admin"),
+            ("org:ci", "org:ci"),
```

Comment:
Did you check with @mitsuhiko to use the `ApiToken` entity?

Reply:
How do we want to deal with the token's length?

Reply:
This is unrelated to this, just adds the new scope generally! This does not add a new token/other stuff yet :)

Reply:
But adding a new scope would imply that other tokens can also use it, do we want this?

Reply:
Yeah, AFAIK we should use the same scopes for both/all types of tokens. We may decide to simply not allow to select this for non-org-based tokens eventually, but IMHO there is no (technical) way it should be forbidden - all access that this scope has can also be achieved via other scopes :)